src/main/java/com/matera/account/account/AccountService.java

```diff
@@ -0,0 +1,56 @@
+package com.matera.account.account;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Service
+public class AccountService {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+    public List<Account> findAll() {
+        return accountRepository.findAll();
+    }
+
+    public List<Account> findClientAccountsByClientId(UUID clientId) {
+        List<Account> allAccounts = accountRepository.findAll();
```

Comment:
here you are fetching every account on the database, this may be a heavy opperation, if we have lots of accounts on the database.

we need to fetch only the client accounts. Spring data works by convention, look at my sugestion on the AccountRepository file, and use it here like:

`eturn accountRepository.findAllByClientId(clientId);`